validator/libs/checks.js

```diff
@@ -71,119 +70,119 @@ var getApiEntityClient = function() {
 
 module.exports = {
   //check if a documentation file exists in a given path
-  docExist: function(fullPath) {
-    var check = false;
+  docExist(fullPath) {
+    let check = false;
     conf.nconf.get('dmv:docFolders').forEach(function(value) {
       try {
         fs.lstatSync(path.join(fullPath, value)).isDirectory();
         if (fileExists(path.join(fullPath, value), 'spec.md') ||
             fileExists(path.join(fullPath, value), 'introduction.md'))
-         check = true;
+         {check = true;}
       } catch (err) {
-
+        console.log('***ERROR*** ' + err);
```

Comment:
introducing the log, you have this errors in the console:
https://travis-ci.org/Fiware/dataModels/jobs/436145121#L560

but this is anyhow reported due to check= false in the final report:
https://travis-ci.org/Fiware/dataModels/jobs/436145121#L1123

Reply:
Fixed - a5c41847